lightwood/encoders/categorical/autoencoder.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, is_target=False):
             self.max_encoded_length = None
         else:
             self.max_encoded_length = 100
-        self.max_training_time = CONFIG.MAX_ENCODER_TRAINING_TIME
+        self.max_training_time = 7200
```

Comment:
this should be in the params for the encoder

Reply:
Hmh, I'll take a look and see if there's a way to add it as a default to the config... but I see no reason why it should be there more so than any of the other encoder attributes. I think we'll end up setting this dynamically based on the total allowed training time anyway.